Code-the-Dream-School/dd-prac-team1-front
# PRA-160 Active links in Navbar

## Active links in Navbar

## Requirements

Navbar should be responsive with active navbar links and hover on them.

## Notes

_Any additional info_

## Test Steps

1) Test the responsiveness of the navbar.
2) When you click outside the mobile navbar, it closes. When you click on the link of the mobile navbar, it closes.
3) You can see hover on the navlinks.
4) We the link is active, it is underlined.

## UI/UX Outcome

<img width="1394" alt="Screen Shot 2023-08-22 at 18 31 47" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/e650ec32-7e68-499c-9c8a-cbca81677bd9">

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
